pages/english.py

```diff
@@ -32,6 +34,9 @@ def main():
         st.session_state.messages.append({"role": "assistant", "content": answer})
         st.chat_message("assistant").write(answer)
 
+        end_time = datetime.datetime.now()
+        print("This query ran for", end_time - start_time, "seconds") # a little performance test
+
```

Comment:
a sample performance test I did. It is of course not necessary and will be removed in the end..